### PR TITLE
Add dummy animal_found and animal_given_away events to fool around wi…

### PR DESCRIPTION
**Pull request by AzisK**
…th some sample data

### CODE REVIEW

File: database/3-fake-data.sql [hunk 1]
```diff
@@ -101,3 +101,20 @@ INSERT INTO animal_event_general
...
+INSERT INTO former_animal_owner
+(name, surname, phone) VALUES
+('Laisvyda', 'Katmeilė', '1234'),
+('Bonifacas', 'Tabletė', '4321'),
+('Konfucijas', 'Šunmylys', '+000999');
+
+INSERT INTO animal_event_given_away
+(former_owner_id, reason, animal_id, date_time) VALUES
+(1, 'Have to go abroad', 1, '2021-01-07'),
+(2, 'Išvyksta į užsieny', 2, '2021-01-01'),
```
Comment (psaulenas) [inline database/3-fake-data.sql:119]: Is it on purpose to mix languages?

**Review by psaulenas — APPROVED** [review]
LGTM

File: database/3-fake-data.sql (hunk 1 above)
Comment (psaulenas) [inline database/3-fake-data.sql:119]:
```suggestion
(2, 'Išvyksta į užsienį', 2, '2021-01-01'),
```

Comment (AzisK) [reply database/3-fake-data.sql:119]: Yes

**Pull request merged**